src/python/gem5/resources/resource.py

```diff
@@ -824,32 +825,37 @@ def obtain_resource(
 ) -> AbstractResource:
     """
     This function primarily serves as a factory for resources. It will return
-    the correct `AbstractResource` implementation based on the resource
+    the correct AbstractResource implementation based on the resource
     requested.
 
     :param resource_name: The name of the gem5 resource as it appears under the
-    "id" field in the `resource.json` file.
+                          "id" field in the ``resource.json`` file.
     :param resource_directory: The location of the directory in which the
-    resource is to be stored. If this parameter is not set, it will set to
-    the environment variable `GEM5_RESOURCE_DIR`. If the environment is not
-    set it will default to `~/.cache/gem5` if available, otherwise the CWD.
-    **Note**: This argument is ignored if the `to_path` parameter is specified.
-    :param download_md5_mismatch: If the resource is present, but does not
-    have the correct md5 value, the resoruce will be deleted and
-    re-downloaded if this value is True. Otherwise an exception will be
-    thrown. True by default.
+                               resource is to be stored. If this parameter is not
+                               set, it will set to the environment variable
+                               ``GEM5_RESOURCE_DIR``. If the environment is not set
+                               it will default to ``~/.cache/gem5`` if available,
+                               otherwise the CWD.
+    			                *Note*: This argument is ignored if the ``to_path``
+    			               parameter is specified.
+    :param download_md5_mismatch: If the resource is present, but does not have
+                                  the correct md5 value, the resoruce will be
```

Comment:
```suggestion
                                  the correct md5 value, the resource will be
```

This is not related to the reformatting, but resource is misspelled as resoruce. I think we can change change this in this PR as it updates the documentation.